### PR TITLE
Enable setting version of python lib via system property

### PR DESCRIPTION
**Pull request by kiendang**
in order to facilitate automatic setting up ScalaPy since it much easier to set a system property than to set an environment variable in Scala.

### CODE REVIEW

**Review by shadaj — CHANGES_REQUESTED** [review]
Looks great! Could you add an item for this to the changelog?

**Comment by kiendang** [comment]
Changelog added. I also switched the order so the system property takes precedence over the env var since the former is more controllable. I can switch back if you prefer the way it was.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ## vNEXT
 ### Highlights :tada:
 + Add support for bracket syntax to facades. The annotation `@PyBracketAccess` can be used on methods to mark them as representing bracket access on an object. The target method must have one (to read the value) or two parameters (to update the value) ([PR #194](https://github.com/shadaj/scalapy/pull/194)).
++ The version of the Python native library can now be controlled with the `scalapy.python.library` system property
```
Comment (shadaj) [inline CHANGELOG.md:5]: Could you link to this PR to be consistent with the other items?

**Review by shadaj — CHANGES_REQUESTED** [review]
Having the property take higher precedence sounds good to me. Just one more change needed to link to this PR in the changelog.

**Review by shadaj — APPROVED** [review]

**Pull request merged**